diff --git a/node_modules/vue/dist/vue.esm-browser.js b/node_modules/vue/dist/vue.esm-browser.js
index 042c78a..db541de 100644
--- a/node_modules/vue/dist/vue.esm-browser.js
+++ b/node_modules/vue/dist/vue.esm-browser.js
@@ -10031,6 +10031,8 @@ function normalizeChildren(vnode, children) {
 }
 function mergeProps(...args) {
   const ret = {};
+  warn$1('warn vue browser')
+  console.log('test vue browser')
   for (let i = 0; i < args.length; i++) {
     const toMerge = args[i];
     for (const key in toMerge) {
diff --git a/node_modules/vue/dist/vue.global.js b/node_modules/vue/dist/vue.global.js
index 65d6acf..642ae2a 100644
--- a/node_modules/vue/dist/vue.global.js
+++ b/node_modules/vue/dist/vue.global.js
@@ -9997,6 +9997,8 @@ Component that was made reactive: `,
   }
   function mergeProps(...args) {
     const ret = {};
+        warn$1('warn vue global')
+    console.log('test vue global')
     for (let i = 0; i < args.length; i++) {
       const toMerge = args[i];
       for (const key in toMerge) {
diff --git a/node_modules/vue/dist/vue.runtime.esm-browser.js b/node_modules/vue/dist/vue.runtime.esm-browser.js
index b78b866..ba19fab 100644
--- a/node_modules/vue/dist/vue.runtime.esm-browser.js
+++ b/node_modules/vue/dist/vue.runtime.esm-browser.js
@@ -9958,6 +9958,8 @@ function normalizeChildren(vnode, children) {
 }
 function mergeProps(...args) {
   const ret = {};
+      warn$1('warn vue esm browser')
+    console.log('test vue esm browser')
   for (let i = 0; i < args.length; i++) {
     const toMerge = args[i];
     for (const key in toMerge) {
diff --git a/node_modules/vue/dist/vue.runtime.esm-bundler.js b/node_modules/vue/dist/vue.runtime.esm-bundler.js
index 928f61f..2edba53 100644
--- a/node_modules/vue/dist/vue.runtime.esm-bundler.js
+++ b/node_modules/vue/dist/vue.runtime.esm-bundler.js
@@ -6,6 +6,7 @@
 import { initCustomFormatter, warn } from '@vue/runtime-dom';
 export * from '@vue/runtime-dom';
 
+console.log('test')
 function initDev() {
   {
     initCustomFormatter();
diff --git a/node_modules/vue/dist/vue.runtime.global.js b/node_modules/vue/dist/vue.runtime.global.js
index 19049ba..c5c7a27 100644
--- a/node_modules/vue/dist/vue.runtime.global.js
+++ b/node_modules/vue/dist/vue.runtime.global.js
@@ -9924,6 +9924,9 @@ Component that was made reactive: `,
   }
   function mergeProps(...args) {
     const ret = {};
+        warn$1('warn vue global')
+    console.log('test vue global')
+    args = []
     for (let i = 0; i < args.length; i++) {
       const toMerge = args[i];
       for (const key in toMerge) {
